src/queries.js

```diff
@@ -129,11 +165,40 @@ function queryByAltText(container, alt) {
 function getByAltText(container, alt) {
   const el = queryByAltText(container, alt)
   if (!el) {
-    throw new Error(`Unable to find an element with the alt text: ${alt}`)
+    const htmlElement = checkDebugLimit(
+      prettyFormat(container, {
+        plugins: [DOMElement],
+        printFunctionName: false,
+        highlight: true,
+      }),
+    )
+    throw new Error(
+      `Unable to find an element with the alt text: ${alt} \nHere is the state of your container: \n${htmlElement}`,
+    )
   }
   return el
 }
 
+function checkDebugLimit(debugContent) {
+  const limit = process.env.DEBUG_PRINT_LIMIT
```

Comment:
What if the limit needed in browser where there's no env, e.g. via Cypress?

I suggest exporting a mutable "defaultOptions" object from the module instead, like Node's util.inspect.defaultOptions.

Reply:
Yes, thats a good point.